### PR TITLE
Scala3 build

### PR DESCRIPTION
**Pull request by sebastian-alfers**

### CODE REVIEW

**Comment by sebastian-alfers** [comment]
Hey @matsluni, any chance I can get your feedback on this? Thanks 🙏

**Comment by matsluni** [comment]
@sebastian-alfers Thanks for your contribution. I'll try to have a look at it soon, currently I am very busy at work.

**Comment by sebastian-alfers** [comment]
@matsluni Thanks!

**Comment by sebastian-alfers** [comment]
Any chance to have this reviewed @matsluni?

**Comment by matsluni** [comment]
Hi @sebastian-alfers, I am currently looking at the code in general and your PR. 

As there are some changes with the license of Akka, I may want to make a release with Akka 2.6 before going to 2.7. I am also seeing, that Scala 2.11 is not supported anymore with more recent Akka versions. Hopefully this can be dropped here as well without issues.

**Comment by sebastian-alfers** [comment]
Feel free ping me if I can be of any help to move this forward, thanks!

File: src/test/scala/com/github/matsluni/akkahttpspi/BaseAwsClientTest.scala [hunk 1]
```diff
@@ -52,21 +52,21 @@ trait LocalstackBaseAwsClientTest[C <: SdkClient] extends BaseAwsClientTest[C] {
...
+  )
+  override val container: GenericContainer = containerInstance
 }
 
 trait ElasticMQSQSBaseAwsClientTest[C <: SdkClient] extends BaseAwsClientTest[C] {
   def service: String
 
   lazy val exposedServicePort: Int = 9324
 
-  override lazy val container: GenericContainer =
```
Comment (sebastian-alfers) [inline src/test/scala/com/github/matsluni/akkahttpspi/BaseAwsClientTest.scala:69]: Scala3 does not like to override a `val` from a trait with a `lazy` declaration.

Comment (matsluni) [reply src/test/scala/com/github/matsluni/akkahttpspi/BaseAwsClientTest.scala:69]: Thanks for the explanation.

File: build.sbt [hunk 2]
```diff
@@ -57,8 +57,8 @@ lazy val root = (project in file("."))
 
 lazy val deps = {
   val awsSDKVersion = "2.11.4"
-  val akkaVersion = "2.6.21"
-  val AkkaHttpVersion = "10.2.10"
+  val akkaVersion = "2.8.1"
+  val AkkaHttpVersion = "10.5.1"
```
[the remark this answers is not in the document]
Comment (sebastian-alfers) [reply build.sbt:61]: Sounds good!

**Comment by matsluni** [comment]
Many thanks for your contribution. 🏅

**Pull request merged**